docs/scenario-dashboard.md

```diff
@@ -0,0 +1,228 @@
+# Batch Scenario Manager
+
+monoDrive's Batch Scenario Manager provides the client with visuals of custom configurations and result viewing for better testing. The application is designed to step through each configuration with pre-built and customizable properties. Once all configurations are made, deploy and run every combination of the properties selected in the simulation. Results show the status of the test after it has run, and keeps different batches organized for review. 
+
+The Batch Scenario Manager can be used as a local application on your computer or through a cloud deployment. The local application enables users to view the test running in the Simulator, while the monoDrive Cloud Solution distributes batch processing to handle high volumes of AV testing jobs. The Cloud Solution is handled through a Kubernetes native application with easy deployment to any private or public cloud.
```

Comment:
I would pull out all references to the cloud deployment from the main document here. Then at the end of this page we can have a separate "Cloud deployment" section containing that overview and a CTA to reach out for more info.